.codespellrc

```diff
@@ -0,0 +1,5 @@
+[codespell]
```

Comment:
this has to stay in `src/tools` with other devtools configurations.

Reply:
well -- codespell would not pick it up automagically when ran from the top directory... 
unfortunately action is not ready for that https://github.com/codespell-project/actions-codespell/issues/67  . Will need to do manual install/execute. Done, force pushed.  Please see more details/reasoning in individual commits if questions arise.